packages/runtime-tools/README.md

```diff
@@ -1,301 +1,28 @@
-# `@module-federation/runtime`
+# `@module-federation/runtime-tools`
 
-- Can be combined with the build plug-in to share basic dependencies according to policies to reduce the number of module downloads and improve the loading speed of modules.
-- Only consume part of the export of the remote module and will not fully download the remote module
-- The runtime calling process can be extended through the module-runtime plug-in mechanism
+Module federation runtime package collection. You can just install it instead of installing all the packages separately. 
```

Comment:
I think we should be able to explain why this package is available. The main purpose of this package is to adapt different tools for implementing module federation

Reply:
Differing tools as well as a easy way to install "synced" package collection. 
No worry about sdk mismatch with runtime etc